scripts/server_client.rb

```diff
@@ -27,7 +28,9 @@ class Client
   include BoltSpec::BoltServer
 
   def initialize
-    @inventory = Bolt::Inventory.new(conn_inventory.merge(easy_config))
+    config = Bolt::Config.default
```

Comment:
The default for this is in the inventory init method, you should only have to initialize and pass in plugins.

Reply:
@lucy I think that config is required for `Bolt::Plugin#setup`

Reply:
@lucywyman ^^ (sorry other lucy :grimacing:  )